tutor/templates/config.yml

```diff
@@ -57,6 +57,11 @@ MONGODB_DATABASE: "openedx"
 MONGODB_PORT: 27017
 MONGODB_USERNAME: ""
 MONGODB_PASSWORD: ""
+MONGODB_REPLICA_SET: ""
+MONGODB_PROTOCOL: "mongodb"
+MONGODB_USE_SSL: ""
```

Comment:
This variable must default to a boolean, as you explained in the docs. And then its usage must reflect the fact that its a boolean (see k8s/deployments.yml and k8s/jobs.yml below).

Reply:
As mentioned above, the Mongo connection to a cluster does not work without the replica set parameter (I saw this empirically and from the [documentation](https://pymongo.readthedocs.io/en/stable/examples/high_availability.html)  in which `replicasSet` needs to be specified in the client)

But I am actually not able to test this anymore to demonstrate that this is correct, because Mongo Atlas has stopped supporting v3.6 clusters (from April 30), and so I am no longer running Mongo on Atlas for Open edX. 

This PR is then only useful when Open edX upgrades to v4.0 - the timing for which seems to be [unclear](https://discuss.openedx.org/t/mongo-4-0-upgrade/3779) - or if anyone runs their own Mongo cluster.

I suggest that we close the PR.

Reply:
@regisb I am actually keen to do the test on lilac - do you have some pointers to running lilac with tutor? I see there is a `lilac` branch in the repo - can I just launch from that?

Reply:
That would be great! Yes, you should install Tutor from the lilac branch. You should apply the first two steps listed here: https://discuss.overhang.io/t/how-to-upgrade-a-tutor-plugin/1488

> * Install Tutor from source, from the release candidate branch: in our particular case, it’s the lilac 4 branch.
> * Build all the tutor images locally (as they are not available from docker.io 1): tutor config save && tutor images build all.

Note that this version already depends on Mongodb 4.0.